api/MediaCapabilities.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
Was this file erroneously included in the PR? It doesn't seem relevant to https://bugzilla.mozilla.org/show_bug.cgi?id=1444139